scapy/plist.py

```diff
@@ -299,10 +306,15 @@ def conversations(self, getsrcdst=None,**kargs):
             except:
                 #XXX warning()
```

Comment:
The comment was here before, I left it but I'm not sure. Do you want me to remove it?